build/windows/installer/conf/fluent-bit-common.conf

```diff
@@ -22,6 +22,20 @@
     Buffer_Size         ${AZMON_FBIT_BUFFER_SIZE}
     Mem_Buf_Limit       ${AZMON_FBIT_MEM_BUF_LIMIT}
 
+[INPUT]
+    Name tail
+    Alias ama-logs_monagent_host_log_tail
+    Tag oms.container.log.flbplugin.ama.*
+    Path C:\\opt\\windowsazuremonitoragent\\datadirectory\\Configuration\\MonAgentHost.1.log
```

Comment:
do we know what's the log size (in terms of # lines )? if its reasonably high, we should do filter only error logs.

Reply:
I think there are atleast 30+ lines...So, yeah will filter for ERROR logs